graphql_api/types/repository/repository.py

```diff
@@ -283,6 +283,15 @@ def resolve_flags_measurements_backfilled(repository: Repository, info) -> bool:
     return dataset.is_backfilled()
 
 
+@repository_bindable.field("isATSConfigured")
+def resolve_is_ats_configured(repository: Repository, info) -> bool:
+    if not repository.yaml or "flag_management" not in repository.yaml:
```

Comment:
If what you're checking is if repository.yaml exists/is not an empty dict, I'd do `if repository.yaml is not None` vs this ^. If you had an empty dict, this evaluates to true `if not {} // True`. Check this out https://stackoverflow.com/questions/3965104/not-none-test-in-python for general rules of thumb

Reply:
Do you feel strongly about this? It seems like it is a matter of preference and I can't see any logical inconsistency between the methods. It looks like `if not {}` returns [False](https://replit.com/@RohitVinnakota2/SelfreliantEmbellishedPolygon#main.py).

Reply:
So this suggestion is mostly since the values of `repository.yaml` can be null by default. Granted, doing `not` on None object will also yield True (cause not None ~ not False = True, see pic below), so I often like to check against the type itself, not strongly about it

<img width="874" alt="Screenshot 2023-12-28 at 10 05 13 AM" src="https://github.com/codecov/codecov-api/assets/82913673/124c98f1-1874-432c-b9e5-619a2c9a0927">